gtclang/CMakeLists.txt

```diff
@@ -89,8 +45,8 @@ make_config_string("Asserts" ${GTCLANG_ASSERTS})
 yoda_report_result("Configuration summary" ${config_info})
 
 # Include the packages (and set the correct, libraries, includes etc.)
-foreach(package bash ccache Python3 Boost clang-format Clang Dawn OpenMP Threads)
```

Comment:
Are Boost and OpenMP direct dependencies here, or are these packages that GridTools depends on?